transit/script/bart.py

```diff
@@ -0,0 +1,396 @@
+from pathlib import Path
+
+import pandas as pd
+import tomllib
+from transit_function import read_dbf_and_groupby_sum, read_transit_assignments
+
+
+def read_nodes(model_run_dir):
+    filepath = Path(model_run_dir) / "nodes.xls"
+    return pd.read_excel(filepath, header=None, names=["Node", "Node Name"])
+
+
+def station_name():
```

Comment:
This probably shouldn't be hardcoded into the script. These are the correspondences to CHAMP node IDs, right? So they could change in the future

Reply:
also, is there a reason this is a function and not just a variable?

Reply:
I just need to match the station's full name with its short abbreviation. So, am I using the full name for mapping? It will be hardcoded, though.

Reply:
Why don't we just make a simple 2-column file with "Station" and "Node"?  These change infrequently enough that they *could* be hard-coded, but this 2-list construction seems error-prone.

Reply:
priority is borderline-👀